setup.cfg

```diff
@@ -55,9 +55,11 @@ install_requires =
     lxml           == 4.9.0  # Must be 4.9.0 for Windows compatibility with prebuilt https://www.lfd.uci.edu/~gohlke/pythonlibs/#lxml
     zope.component ~= 5.0.1
     zope.interface ~= 5.4.0
-    pds.api-client ~= 1.1.0
```

Comment:
Hi @nutjob4life , so you stopped using the api client 😢 ?
We'll see later if we can re-integrated that or the upper layer. I guess it was too unstable and you missed that /all feature.

Since we have that demo scheduled tomorrow with the nodes I will do the release now with the PR merged.

Thanks

Reply:
Sorry @tloubrieu-jpl! The client doesn't support the "/latest" or "/all" endpoints which I need to use. But the server itself is easy enough to take advantage of, so Python Requests works out great 🎉